Strange error message from LibraryImport

Compiling:
```C#
using System.Runtime.InteropServices;

Console.WriteLine(Interop.MyFunction());

partial class Interop
{
    [LibraryImport(Abc)]
    internal static partial int MyFunction();
}
```
errors with the message:
"The 'Invalid syntax' configuration is not supported by source-generated P/Invokes."

That's a very strange error message :smile:

Do we even need a diagnostic in this case?  It's not going to compile, anyway, as the C# compiler itself also issues "error CS0103: The name 'Abc' does not exist in the current context".

--- a/src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/LibraryImportGenerator.cs
+++ b/src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/LibraryImportGenerator.cs
@@ -274,7 +274,6 @@ namespace Microsoft.Interop
 
             if (libraryImportData is null)
             {
-                generatorDiagnostics.ReportConfigurationNotSupported(generatedDllImportAttr!, "Invalid syntax");
                 libraryImportData = new LibraryImportData("INVALID_CSHARP_SYNTAX");
             }
 

--- a/src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CompileFails.cs
+++ b/src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CompileFails.cs
@@ -149,9 +149,9 @@ namespace LibraryImportGenerator.UnitTests
         {
             yield return new object[] { CodeSnippets.RecursiveImplicitlyBlittableStruct, 0, 1 };
             yield return new object[] { CodeSnippets.MutuallyRecursiveImplicitlyBlittableStruct, 0, 2 };
-            yield return new object[] { CodeSnippets.PartialPropertyName, 1, 2 };
-            yield return new object[] { CodeSnippets.InvalidConstantForModuleName, 1, 1 };
-            yield return new object[] { CodeSnippets.IncorrectAttributeFieldType, 1, 1 };
+            yield return new object[] { CodeSnippets.PartialPropertyName, 0, 2 };
+            yield return new object[] { CodeSnippets.InvalidConstantForModuleName, 0, 1 };
+            yield return new object[] { CodeSnippets.IncorrectAttributeFieldType, 0, 1 };
         }
 
         [Theory]

